src/main/java/com/google/sps/managers/DatastoreManager.java

```diff
@@ -178,7 +178,7 @@ public Stage retrieveStage(String stageID) {
       return null;
```

Comment:
This isn't an issue but you can just do return; without putting null and it does the same thing.

Reply:
Modified, thanks.